src/CMakeLists.txt

```diff
@@ -17,5 +18,4 @@ add_subdirectory(init)
 add_subdirectory(ipc)
 add_subdirectory(lib)
 add_subdirectory(math)
-add_subdirectory(platform)
```

Comment:
Sorry, but I don't understand this change. Why do you remove this here?

Reply:
If left in this position platform is gone through as last file and the arch needs platform defines for linker/asm scripts.
Tldr: Won't build otherwise.

Reply:
This order shouldn't matter for build. Can you give some more details of what's needed by arch ?

Reply:
It does since elements that are under platform sub directory are used in ASM files that won't build unless these  defines are run first.

Reply:
Will be fixed in separate PR for #1235